vernac/library.mllib

```diff
@@ -7,7 +7,6 @@ Global
 Decl_kinds
```

Comment:
Is moving this file intended?

Reply:
Yes, this is already done in one of the proof PRs.

Reply:
Required quite a bit of untangling.

Reply:
I mean moving the mllib in this PR, not decl_kinds in the future.

Reply:
Ah no, that was mistake sorry.